setup.py

```diff
@@ -1,6 +1,7 @@
 from setuptools import setup
```

Comment:
I'm surprised that we don't need more here, but perhaps because it all now lives in `pyproject.toml`?